themes/default/content/learn/embedding-pulumi/considerations/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Understanding Extra Considerations"
```

Comment:
Minor thing, but this reads like "extra considerations" are a kind of Pulumi-ism. I don't have a great suggestion for how to address it, but since it'll end up as the H1 of the page, and these pages are so well indexed, it might be a good place to use more relevant keywords, like "logging" or "exceptions", maybe.

Reply:
Hmm... good point.

Reply:
Fixed